@ndlib/gatsby-plugin-marble-elasticsearch/package.json

```diff
@@ -13,7 +13,7 @@
   "license": "APACHE2",
   "dependencies": {
     "aws-sdk": "^2.916.0",
-    "elasticsearch": "^16.7.2",
+    "elasticsearch": "^15.0.0",
```

Comment:
why are we going down here?  It seems like we should stay as high as possible.

Reply:
Not sure what happened here. Removing change